### PR TITLE
reshuffle doc content and fix some typos

### PR DESCRIPTION
**Pull request by tobru**
## Checklist

- [X] Keep pull requests small so they can be easily reviewed.
- [X] Update the documentation.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/deployment.adoc [hunk 1]
```diff
@@ -4,28 +4,18 @@ The Lieutenant API is meant to be deployed on a Kubernetes cluster. It needs to
...
 [source,shell]
 ----
 kubectl create ns $namespace
 
-kubectl -n $namespace apply -k deploy/
+kubectl -n $namespace apply -k github.com/projectsyn/lieutenant-api/deploy
 ----
 
-
+[discrete]
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/deployment.adoc:16]: The section heading is in order. So what is the reasoning for this? If it is to suppress the table of content, then do so in the document attributes.

**Review by corvus-ch — COMMENTED** [review]
My comments might be out of scope for this change. Feel free to leave them for later.

File: docs/modules/ROOT/pages/how-tos/deployment.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/how-tos/deployment.adoc:16]: I learned that this is the way to suppress the table of content in Asciidoc, all other means I tried didn't work on our setup and I already lost quite some time with it.

Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/deployment.adoc:16]: That is a pity.

**Pull request merged**